docs/guides/basics/create_new.md

```diff
@@ -4,19 +4,31 @@ repositories.
 
 To create a new Project, visit [www.floydhub.com/projects](https://www.floydhub.com/projects) and click on the "New Project" button on the top right hand corner.
 
-![Create new project](../../img/create_new_project.jpg)
+![Create new project](../../img/create_project/create_project.png)
 
-Give the project a name and an apt description.
+Give the project a name and a description.
+
+The `Owner` field defines the user space under which this project belong. If you are inside a Team select the name of your Team to create the project under the Team space.
 
 The `Visibility` field indicates who can see your project. If you set it to `Public`, anyone can see your project, your code and data. If you are working on an open source project, this is a great way to share and contribute to the FloydHub community. If your code or data is proprietary, please select `Private`. This will ensure that only you and your team will have access to this project.
 
-Once you have created a Project, you can start running jobs using the [floyd run](../../commands/run) command. For example, to start a Jupyter Notebook job:
+Once you have created a Project, you can start running jobs using the [floyd run](../../commands/run) command. For example, to start a CLI Job:
+
+```bash
+$ floyd init alice/quick-start
+Project "quick-start" initialized in the current directory
+
+$ floyd run --gpu --env tensorflow-1.9 'python train.py'
+Syncing code ...
+```
+
+Let's make another example where Alice is a member of the `Wonderland` Team and want to run the same CLI Job of above inside the Team space:
```

Comment:
Here is another example where Alice is a member of the `wonderland` team and wants to run the same CLI Job but in her team namespace: